docs/en/integrations/dbt/dbt-limitations.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: Limitations
+sidebar_position: 8
+description: Limitations of the dbt ClickHouse plugin
+---
+
+# Limitations
+
+The current ClickHouse plugin for dbt has several limitations users should be aware of:
+
+
+
+1. As noted in [Creating an Incremental Materialization](./dbt-incremental-model#internals), incremental changes are currently loaded into an in-memory table. ClickHouse does not provide distribution for this table, so changesets must not be larger than the memory of the orchestrator node receiving the query. Users should either avoid using the plugin on datasets where changes/additions are very high or schedule this operation to run at appropriate intervals to ensure all changes can be captured within memory limits. As noted[ here](https://clickhouse.com/docs/en/engines/table-engines/special/memory), we recommend that in-memory table engines should not exceed 100,000,000 rows. Use this as an upper bound and schedule dbt run’s accordingly, i.e., ensure that the execution interval is frequent enough to not exceed this limit.
```

Comment:
for readability, can you replace:
should not exceed 100,000,000 rows.
by
should not exceed 100 Million rows.